README.md

```diff
@@ -54,14 +54,18 @@ GitHub Enterprise repositories are not supported.
 
 ![image](/images/install02.png)
 
-3) Then click "Add tool".
+3) Click on the button "Configure scanning tool"
```

Comment:
I think whether the user sees "Add tool" vs "Configure scanning tool" depends on whether or not you already use a Code Scanning tool, such as CodeQL.